tools/accuracy_checker/accuracy_checker/utils.py

```diff
@@ -31,14 +31,10 @@
 from collections.abc import MutableSet, Sequence
 from io import BytesIO
 
+import defusedxml.ElementTree as et
```

Comment:
please wrap to unsupprted package for avaoid situation, when this package is not installed (by default we install only core dependency on yocto) or move to core-requirements (as I know VOC format converter uses et and it should continue work on yocto)